### PR TITLE
update warning message in readme about pr guidelines

### PR DESCRIPTION
**Pull request by ryancbahan**
## Description

Screen Shot 2021-09-03 at 11 41 57 AM

Updating the readme to reflect our current position on adding new packages to quilt.

## Type of change

Update to readme. 

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by ryancbahan** [comment]
@atesgoral I ran `yarn generate docs` after CI was yelling at me, and now it looks like I don't have any changes to the readme in this branch. Curious if I might've borked something here 😓

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ A loosely related set of packages for JavaScript/TypeScript projects at Shopify.
 
 These libraries compose together to help you create performant modern JS apps that you love to develop and test. These packages are developed primarily to be used on top of the stack we like best for our JS apps; Typescript for the flavor, Koa for the server, React for UI, Apollo for data fetching, and Jest for tests. That said, you can mix and match as you like.
 
-⚠️ Over the past few years, this repo has become a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. For this reason, we are no longer accepting new packages.
+⚠️ Before submitting a pull request, please speak with the Web Foundations team on guidance as to whether a package might belong in quilt.
```
Comment (atesgoral) [inline README.md:13]:
```suggestion
⚠️ Over the past few years, this repo has become a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. Before submitting a pull request, please speak with the Web Foundations team on guidance as to whether a package might belong in quilt.
```

File: templates/ROOT_README.hbs.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ A loosely related set of packages for JavaScript/TypeScript projects at Shopify.
 
 These libraries compose together to help you create performant modern JS apps that you love to develop and test. These packages are developed primarily to be used on top of the stack we like best for our JS apps; Typescript for the flavor, Koa for the server, React for UI, Apollo for data fetching, and Jest for tests. That said, you can mix and match as you like.
 
-⚠️ Over the past few years, this repo has become a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. For this reason, we are no longer accepting new packages.
+⚠️ Before submitting a pull request, please speak with the Web Foundations team on guidance as to whether a package might belong in quilt.
```
Comment (atesgoral) [inline templates/ROOT_README.hbs.md:13]:
```suggestion
⚠️ Over the past few years, this repo has become a dumping ground for a variety of packages unrelated to the core problems Quilt aims to solve. Before submitting a pull request, please speak with the Web Foundations team on guidance as to whether a package might belong in quilt.
```

**Comment by atesgoral** [comment]
Chatting with the team, we decided it would be best to preserve the first sentence for more context.

**Comment by ryancbahan** [comment]
Makes sense to me!

**Pull request merged**